recipe/post-link.bat

```diff
@@ -0,0 +1,7 @@
+@echo off
```

Comment:
This shouldn't be needed. The install --prefix should still work, it's just not cross-platform. We may have to revert noarch, but we *shouldn't* have to revert removing this problematic post-link.

Reply:
I get it. do you think it is related to the failures thought?

Reply:
Not that particular failure, at least. I I think that's a weird sensitivity in the test. Hopefully relaunching the test will fix it.

Reply:
I reliably get the fail on test_async. Unfortunately, it is hard to use `pyargs` and `--ignore` or `-k` on windows... i'm looking into the simplest way to delete the `test_async.py` from the installed package...

Reply:
I think we can add it to a line in the script section if you can tell me what the command is.

I'm kinda half on vacation now, so feel free to open a second PR yourself.

Reply:
well, now pypi is down, so not building much of anything... but yeah, i'll open a second pr.